### PR TITLE
Add new line at the end of istio.RELEASE

### PR DESCRIPTION
**Pull request by chxchx**
**Release note**:

```release-note
none
```

### CODE REVIEW

**Review by ldemailly — APPROVED** [review]
thanks!

File: toolbox/githubctl/main.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ func CreateIstioReleaseUploadArtifacts() error {
 			istioRepo, releaseTag, archiveDir); err != nil {
 			return err
 		}
-		if err := u.WriteFile(releaseTagFile, *nextRelease); err != nil {
+		if err := u.WriteFile(releaseTagFile, *nextRelease+"\n"); err != nil {
```
Comment (ldemailly) [inline toolbox/githubctl/main.go:207]: you could also use Fprintln https://golang.org/pkg/fmt/#Fprintln

**Comment by ldemailly** [comment]
typo on line 214
 		"Create release %s on istio, update next release tag and download scrpit", releaseTag)

scrpit vs script

and can we not do the download script ?

File: toolbox/githubctl/main.go [hunk 2]
```diff
@@ -204,7 +204,7 @@ func CreateIstioReleaseUploadArtifacts() error {
...
 			return err
 		}
 		return u.UpdateKeyValueInFile(
```
Comment (ldemailly) [inline toolbox/githubctl/main.go:210]: that line should be flag based or just removed

**Comment by sebastienvas** [comment]
Sorry I merged this too soon

**Comment by ldemailly** [comment]
I was too slow to read the rest of the pr / or too fast to accept it :-) no problem

**Comment by chxchx** [comment]
Please take a look at https://github.com/istio/test-infra/pull/448 which finishes this premature PR.

**Pull request merged**